src/components/Main/AgentInput/index.js

```diff
@@ -18,17 +20,75 @@ class AgentInput extends Component {
       : this.setState({ relWords: value })
   }
 
-  handleSubmit = (e) => {
+  handleSendEvent(e) {
     e.preventDefault();
+      this.setState({
+      currentChatMessage: ''
+    });
+  }
+  renderChatLog() {
+    return this.state.chatLogs.map((chat) => {
+      return (
+        <li key={`chat_${chat.id}`}>
+          <span className='chat-message'>{ chat.content }</span>
+          <span className='chat-created-at'>{ chat.created_at }</span>
+        </li>
+      );
+    })}
+    handleChatInputKeyPress(event) {
+      if(event.key === 'Enter') this.handleSendEvent(event);
+    }
+
+  updateCurrentChatMessage({target}) {
+      this.setState({
+        currentChatMessage: target.value
+      });
+  }
+    createSocket() {
+      // let cable = Cable.createConsumer(`ws://localhost:3000/cable/${this.props.token}`);
+      // this.chats = cable.subscriptions.create({
+      //   channel: 'GameDataChannel'
+      // }, {
+      //   connected: () => {},
+      //   received: (data) => {
+      //     let chatLogs = this.state.chatLogs;
+      //     chatLogs.push(data);
+      //     this.setState({ chatLogs });
+      //   },
+      //   create: function(chatContent) {
+      //     this.perform('create', {
+      //       content: chatContent
+      //     });
+      //   }
+      // });
+    }
+  componentWillMount() {
+    this.createSocket();
   }
 
   render() {
     return (
       <form className="AgentInput">
-        <ConversationsList name="hint" type="text" value={this.state.hint} onChange={this.handleChange} />
         {/* <input name="hint" type="text" value={this.state.hint} onChange={this.handleChange} /> */}
         <input className="num-input" name="relWords" type="number" value={this.state.relWords} onChange={this.handleChange} />
         {/* <input type="submit" onClick={this.handleSubmit} /> */}
+        <h1>Chat</h1>
+            <ul className='chat-logs'>
+              { this.renderChatLog() }
+            </ul>
+            <input
+              onKeyPress={ (e) => this.handleChatInputKeyPress(e) }
```

Comment:
`onKeyPress` listening to `Enter`key, would we want to handle this `onSubmit` of the `form` ?
I don't know too much about this, or whether it needs to be done the way it is?? 🤷‍♂

Reply:
It could be either way really. Up to the front end to decide